tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1003 @@
+-  A sample interface configuration is provided below  
+for reference:  
+
+```
+<insert configuration code snippet here>
+## Creating the base VPN gateway configuration
+
+Follow this multi-step procedure to create the base VPN configuration. 
+```
+
+<Eliminate any steps that are not required by the `<vendor name><product name>` device.>    
+
+### Configure the IKEv2 proposal and policy
+
+<Insert the instructions for creating the IKEv2 proposal and policy here. Below are some examples of IKE algorithms to specify as part of the instructions.>
+
+-  **Encryption algorithm** – <list required algorithms here>
+-  **Integrity algorithm** – <list required algorithms here>
+-  **Diffie-Hellman group **– <list required group here>
+```
+<insert configuration code snippet here>
+### Configure the IKEv2 keyring
+
+<Insert the instructions for creating the IKEv2 keyring here.>
+
+<insert configuration code snippet here>
+```
+
+### Configure the IKEv2 profile
+
+<Insert the instructions for creating the IKEv2 profile here. Below are some examples of parameters to set.>  
+
+-  `IKEv2 Lifetime` - Set the lifetime of the security associations, after which a reconnection occurs. Set to 36,000 seconds as recommended configuration on a <product name> router.
+-  `DPD`** **– Set the dead peer detection interval and retry interval, if there are no response from the peer, the SA created for that peer is deleted. Set to 60 seconds keepalive interval and 5 seconds retry interval as recommended configuration on a <product name>router.  
+
+```
+<insert configuration code snippet here>
+### Configure the IPsec Security Association (SA)
+
+<Insert the instructions for creating the IPsec SA here. Below are some examples of parameters to set.>
+```
+
+-  `IPsec SA lifetime` – 1 hour is the recommended value on ASR 1000 router.
+-  `IPsec SA replay window-size` – 1024 is the recommended value on ASR 1000 router.
+```
+<insert configuration code snippet here>
+### Configure the IPsec transform set
+
+<Insert the instructions for creating the IPsec transform set here.>
+```
+
+`<insert configuration code snippet here>`  
+
+### Configure the IPsec profile
+
+<Insert the instructions for creating the IPsec profile here. Below are some examples of parameters to set.>  
+
+-  Perfect Forward Secrecy (PFS) - PFS ensures that the same key will not be generated again, so forces a new diffie-hellman key exchange. Set to group16 as recommended configuration on ASR 1000 router.
+-  SA Lifetime - set the lifetime of the security associations (after which a  
+  reconnection will occur). Set to `3600 seconds` as recommended configuration  
+  on ASR 1000 router.
+```
+<insert configuration code snippet here>
+### Configure the IPsec static virtual tunnel interface (SVTI)
+
+<Insert the instructions for creating the IPsec SVTI here. Below are some examples of parameters to set.>
+```
+
+-  Adjust the maximum segment size (MSS) value of TCP packets going through a  
```

Comment:
these next few lines are displaying as separate lines instead of part of the bullet.

Reply:
Still need to fix.